### PR TITLE
update Text field component to link Label ("for" attribute) with input's id

### PR DESCRIPTION
**Pull request by phucduong86**

### CODE REVIEW

File: src/components/textfield/TextField.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ export default class TextFieldComponent extends BaseComponent {
   elementInfo() {
     const info = super.elementInfo();
     info.type = 'input';
-
+    info.attr.id = "".concat(this.component.key);
```
Comment (randallknutson) [inline src/components/textfield/TextField.js]: Is there a reason you are doing a concat with an empty string? If you want ensure it is a string you could use this.component.key.toString() although it should already be a string.

Comment (phucduong86) [reply src/components/textfield/TextField.js]: No it works without the concat. 
I saw this done somewhere else in the code base and just thought there was a reason for it.

**Comment by phucduong86** [comment]
Thanks Randall I just updated the line.

**Comment by travist** [comment]
Thanks for the contribution @phucduong86

**Pull request merged**